src/_nebari/upgrade.py

```diff
@@ -1235,6 +1236,81 @@ def _version_specific_upgrade(
             )
             rich.print("")
 
+        rich.print("\n ⚠️ Upgrade Warning ⚠️")
+
+        text = textwrap.dedent(
+            """
+            Nebari version [green]2024.9.1[/green] introduces changes to how group
+            directories are mounted in JupyterLab pods, now only groups with specific
+            permissions will have their directories mounted.
+
+            You will be asked to confirm how Nebari should handle your groups.
+            No data will be lost during this operation. You can reverse this at any time
+            by adding or removing the `allow-group-directory-creation-role` from your
+            groups in the Keycloak UI.
+
+
+            For more information, please see the [green][link=https://www.nebari.dev/docs/how-tos/group-directory-creation]documentation[/link][/green].
+            """
+        )
+        rich.print(text)
+
+        confirm = Prompt.ask(
+            "[bold]Would you like Nebari to update your group permissions now?[/bold]",
```

Comment:
If we are clear with what the feature is actually doing in the intro text, then I think we can change this to:

Would you like Nebari to assign `allow-group-directory-creation-role` to all of your groups now?